pkgs/applications/misc/appmenu-gtk-module/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, fetchurl, cmake, glib, gtk2, gtk3 }:
+let
+xinitfile = fetchurl {
+      url = "https://raw.githubusercontent.com/archlinux/svntogit-community/packages/appmenu-gtk-module/trunk/80-appmenu-gtk-module";
+      sha256 = "1g4zjqnr89achb8k97j26f0dl8wy7afm4dbdmmq60mdqlx76q02c";
+};
+in
+stdenv.mkDerivation {
+  pname = "appmenu-gtk-module";
+  version = "0.7.3.2";
+  src = fetchFromGitHub {
```

Comment:
Looks like this is just a mirror of GitLab repo: https://gitlab.com/vala-panel-project/vala-panel-appmenu

Reply:
I've change it to fetch from git.